src/main/java/com/google/sps/servlets/WaitParent.java

```diff
@@ -0,0 +1,15 @@
+package com.google.sps.servlets;
+
+import java.util.Date;
+import java.util.List;
+
+// Helper class to hold dates and times lists
+public class WaitParent {
```

Comment:
`WaitParent` is kind of a confusing name. What is it the parent of. Maybe a better name could be `WaitData`